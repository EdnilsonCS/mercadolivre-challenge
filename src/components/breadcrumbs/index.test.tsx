import { composeStories } from '@storybook/testing-react';

import {
  render, screen,
} from '@testing-library/react';
import * as stories from './index.stories';

const {
  BreadcrumbsDefault,
} = composeStories(stories);

describe('Test for card component', () => {
  it('should match snapshot in default style', () => {
    const { container } = render(<BreadcrumbsDefault />);

    expect(container).toMatchSnapshot();
  });

  it('should have a text value in breadcrumb', () => {
    render(<BreadcrumbsDefault />);

    const findText = screen.getByText('Iphone > Smarthphone > 32GB');

    expect(findText).toBeTruthy();
  });
});
