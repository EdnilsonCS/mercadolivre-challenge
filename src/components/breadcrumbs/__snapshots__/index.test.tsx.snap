// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for card component should match snapshot in default style 1`] = `
<div>
  <div>
    <span>
      Iphone &gt; Smarthphone &gt; 32GB
    </span>
  </div>
</div>
`;
